pyproject.toml

```diff
@@ -40,6 +40,10 @@ openai = [
     "tiktoken~=0.4.0",
 ]
 
+ctransformers = [
+    "ctransformers~=0.2.14",
```

Comment:
Can you also add this to `requirements.txt`? (the doc build requires all deps to be installed because it imports the modules for autodoc)

Reply:
Added to requirements.txt